File: Coder/paiza/Challenge/Rank_D/_47.java
import java.util.Scanner;

public class _47 {
	public static void main(String[] args){
		try(Scanner in = new Scanner(System.in)){
			int n = in.nextInt();
			String a = n % 2 == 0 ? "even" : "odd";

			System.out.println(a);
		}
	}
}

File: Coder/paiza/Challenge/Rank_D/_47.hs

main :: IO ()
main = do
  n <- readLn
  putStrLn $ if odd n then "odd" else "even"

{- 問題にHaskellで望んだものの
   回答可能言語にHaskellがなかった時の心情を答えよ
-}
